benchmark/blob/blob-text.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+const common = require('../common.js');
+const { Blob } = require('buffer');
+
+const bench = common.createBenchmark(main, {
+  bytes: [0, 8, 128],
+  n: [1e6],
+  operation: ['text', 'arrayBuffer', 'stream']
+});
+
+async function run(n, bytes, operation) {
+  const buff = Buffer.allocUnsafe(bytes);
+  const source = new Blob(buff);
+  bench.start();
+  for (let i = 0; i < n; i++) {
+    switch (operation) {
+      case 'text':
+        await source.text();
+        break;
+      case 'arrayBuffer':
+        await source.arrayBuffer();
+        break;
+      case 'stream':
+        source.stream();
```

Comment:
Keep in mind that this doesn't properly compare to the other operations without actually consuming the data – but I'd leave it up to you whether you want to include that or not. Without it, this part of benchmark isn't very meaningful, it measures stream creation time rather than data reading time, but it does compare meaningfully across versions in that way.

Reply:
Yes, promisifying this is the correct approach. I'll update this in a new pull request, and for now just focus on opening similar pull requests and reducing the ticks.